### ISSUE TITLE
Dolphin crashes symbol lookup error: searpc_create_named_pipe_client

### ISSUE THREAD

**Issue by ghost** [Skycoder42/dolphin-seafile-plugin#5]
Dolphin crashes with the following message, when you create an empty .seafile-data :
```
qt.accessibility.core: Cannot create accessible child interface for object:  PlacesView(0x561bec7e8c10)  index:  14  
dolphin: symbol lookup error: /usr/lib/x86_64-linux-gnu/qt5/plugins/libfileviewseafileplugin.so:  undefined symbol: searpc_create_named_pipe_client  
```

When you go into the seafile main directory where the non-empty .seafile-data dir is located and enable show hidden files, it crashes with the same error, but not unless you enable show hidden files.

libsearpc-dev version: 3.0.8-6

**Comment by Skycoder42** [comment]
Well it means that it can't find that method - try explicitly setting the `LD_LIBRARY_PATH` environment variable to see if that fixes the problem.
